### PR TITLE
[Fix] Show uncaught exception in frontend and exit heroic.

### PR DESCRIPTION
**Pull request by Nocccer**
- According to #1118 this can fill the logs.
- Also uncaught exception should not be ignored and instead shown as message box in the frontend.
- Close the app to avoid unexpected behavior. We should catch all errors we know can be thrown.

Can be tested by throwing a error somewhere in the backend.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by flavioislima — APPROVED** [review]
NIce! This is pretty good to have 👍🏽

File: electron/main.ts [hunk 1]
```diff
@@ -471,8 +471,22 @@ ipcMain.on('frontendReady', () => {
 })
 
 // Maybe this can help with white screens
```
Comment (CapsAdmin) [inline electron/main.ts:473]: What white screens are being mentioned here?

Comment (flavioislima) [reply electron/main.ts:473]: I think was an old Electron bug that sometimes the app became blank in some setups, this was a general issue not only in Heroic so I found this solution when searching on the web.
Not sure if they fixed it already.

**Comment by Nocccer** [comment]
I want to have feedback from the issue i linked but he don't repsonse. We can merge it or not anyway. Opinions?

**Pull request merged**